src/platform/AT32/dshot_bitbang.c

```diff
@@ -583,7 +583,7 @@ static void bbUpdateComplete(void)
     }
 
 #ifdef USE_DSHOT_CACHE_MGMT
```

Comment:
This code is  H7 only

Reply:
Yes it's been copied into the AT driver... I'll remove it.